script/github-actions/filter-files-changed.js

```diff
@@ -0,0 +1,12 @@
+const args = process.argv.slice(2);
+const FILES = args[0].slice(1, -1).split(','); // remove unnecessary characters
+let lintFiles = '';
+
+FILES.forEach(file => {
+  const fileExt = file.split('.').pop();
+  if (fileExt === 'js' || fileExt === 'jsx') {
+    lintFiles += `${file} `;
+  }
+});
+
+console.log(`::set-output name=FILES::${lintFiles}`); // eslint-disable-line no-console
```

Comment:
What say you to doing this more succinctly?

```suggestion
const filteredFiles = files.filter(file => /.+\.jsx?/.test(file)).join(' ');
console.log(`::set-output name=FILES::${filteredFiles}`); // eslint-disable-line no-console
```